tests/ert_tests/gui/test_gui_load.py

```diff
@@ -5,6 +5,7 @@
 import pytest
 import qtpy
 from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget
```

Comment:
Are there restrictions in the name imports per line?
`from qtpy.QtCore.Qt import Key_Down, Key_Backspace, Key_Enter, LeftButton` would limit the amount of stuff imported together with Qt, but also would be on the limit of readability. Do you know about guidelines on that?

Reply:
No restrictions, I think general readability of the entire source code file should matter on this. I would like to keep e.g. `Qt.Key_Down` as is, as it tells me that `Key_Down` is something Qt-specific.